ruff.toml

```diff
@@ -0,0 +1,42 @@
+target-version = "py39"
+line-length = 120
+exclude = [
+  ".git,",
+  "__pycache__",
+  "build",
+  "sunkit_spex/version.py",
+  "sunkit_spex/legacy/fitting_legacy"
+]
+
+[lint]
+select = ["E", "F", "W", "UP", "PT"]
+extend-ignore = [
+  # pycodestyle (E, W)
+  "E501", # LineTooLong                             # TODO! fix
```

Comment:
What's this?

Same for lines below marked `# TODO! fix`

Reply:
Eh ignore line too long, pytext fixture as tuples and purest raise too broad errors until they are fixed in the code base as they can't be automatically fixed